### ISSUE TITLE
Exclude categories

### ISSUE THREAD

**Issue by HitmanJ** [nyaadevs/nyaa#27]
Hello there.
It'd be nice to be able to exclude certain categories from my searches as well from the entire website. Former Nyaa had this option on our profiles, so I think we should have it back there now, since it's such an useful option.

Thanks for your hard work.

**Comment by stvkno** [comment]
Agree - BUT it should not use javascript to filter categories

when pulling torrent results from the database, it should exclude the categories in the query so that a full page of results will be returned instead of getting  a page with 5 results instead of the full 25 or 30 or whatever it was on nyaa.

**Comment by ohbejuan** [comment]
I agree, I like the way Nyaa let me pick the catagories that show up on the main page. I really don't need non english subs.

**Comment by Entangled-Cable** [comment]
Any progress on this? I really would prefer not having to select the same filter every time I search for something. This is like the only feature I registered for.

**Comment by sharkykh** [comment]
IMO it would be easier once we merge #254. There are some stuff that need to merge before this, though.

**Comment by Xesucrist0** [comment]
I came for this. I hope to see this implemented soon.
